addons/website/views/snippets/s_cta_card.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_cta_card" name="Card Call to Action">
+    <section class="s_cta_card o_cc o_cc2 pt64 pb64" data-oe-shape-data="{'shape':'web_editor/Blocks/03'}">
+        <div class="o_we_shape o_web_editor_Blocks_03"/>
+        <div class="container">
+            <div class="row align-items-center">
+                <div class="col-lg-9">
```

Comment:
`XL`is ok, but `LG` is suboptimal.
Use a `col-lg-8` - `col-lg-4`columns combination to make the card breath a bit 😉

| this pr | proposal |
|--------|--------|
| <img width="1012" alt="image" src="https://github.com/user-attachments/assets/d1ba866a-9a64-42c4-860e-cf1cf48d9c7f"> | <img width="1016" alt="image" src="https://github.com/user-attachments/assets/d40985c0-6a28-4c3b-be8d-940fab1c7563"> |